examples/application/main.go

```diff
@@ -72,4 +72,49 @@ func main() {
 	} else {
 		log.Printf("API authorized successfully.")
 	}
+
+	// Generate a login flow.
+	prompt := "user_prompt"
```

Comment:
Shall we add an example prompt here?

Reply:
Btw, I think we can organize the examples/ directory into a set of examples for each category. Having all examples related to applications in a single file wouldn't gonna serve the purpose of having examples IMO.

Eg: 

- examples
   - applications
     - generate-login-flow-by-prompt.go

WDYT @malithie

Reply:
Yes I also thought the same. Will refactor separately.

Reply:
+1

Reply:
Added an example prompt with https://github.com/asgardeo/go/pull/9/commits/8e3395615ecd7b5b5c983427616dc726013b98b3